Sources/Markdown/Structural Restrictions/BasicBlockContainer.swift

```diff
@@ -21,4 +24,14 @@ extension BasicBlockContainer {
     public init(_ children: BlockMarkup...) {
         self.init(children)
     }
+
+    /// Create this element with a sequence of block markup elements, and optionally inherit the source range from those elements.
+    public init(_ children: BlockMarkup..., inheritSourceRange: Bool) {
+        self.init(children, inheritSourceRange: inheritSourceRange)
+    }
+
+    /// Default implementation of `init(_:inheritSourceRange:)` that discards the `inheritSourceRange` parameter.
+    public init<Children: Sequence>(_ children: Children, inheritSourceRange: Bool) where Children.Element == BlockMarkup {
```

Comment:
Weren't `some` and `any` added in a relatively recent Swift release? What's our minimum supported Swift version for Swift-Markdown?

Reply:
There were many proposals for the `some` keyword ([SE-0328][], [SE-0341][], [SE-0360][]) and `any` keyword ([SE-0335][], [SE-0352][], [SE-0353][]) but AFAICT they're all supported as of Swift 5.7 which is the minimum version that Swift-Markdown currently supports.

[SE-0328]: https://github.com/swiftlang/swift-evolution/blob/main/proposals/0328-structural-opaque-result-types.md
[SE-0341]: https://github.com/swiftlang/swift-evolution/blob/main/proposals/0341-opaque-parameters.md
[SE-0360]: https://github.com/swiftlang/swift-evolution/blob/main/proposals/0360-opaque-result-types-with-availability.md

[SE-0335]: https://github.com/swiftlang/swift-evolution/blob/main/proposals/0335-existential-any.md
[SE-0352]: https://github.com/swiftlang/swift-evolution/blob/main/proposals/0352-implicit-open-existentials.md
[SE-0353]: https://github.com/swiftlang/swift-evolution/blob/main/proposals/0353-constrained-existential-types.md

Reply:
I just ran a test using a Swift 5.7 Docker image and it seemed to build just fine, so i went ahead and added it. 🤷‍♀️

Reply:
I think you could do the same [in](https://github.com/swiftlang/swift-markdown/pull/202/files#diff-d97a4b8aa26b46c68bb2b77a503186dc5d5f04ef60911b4ceda6536ab6e00b64R72) [these](https://github.com/swiftlang/swift-markdown/pull/202/files#diff-f59c5b4ca17e9d985a228961df6d1fd84de1796ea610790d2782b67617b8cb57R36) [places](https://github.com/swiftlang/swift-markdown/pull/202/files#diff-5d384b654f21382a22640439cdee6651454830012c05b1b7958e35a001cb1468R36) [as well](https://github.com/swiftlang/swift-markdown/pull/202/files#diff-a0b8a929f46f0151ab7c1d1eb7925b931f78f3f31b8ed1770e515346dbe96a0fR17).

Reply:
Right, the reason this code was written like this was because of the way the original protocol requirement was written. If we want to do this conversion in one place we should do it everywhere. 😅

Reply:
And now that i'm looking through the rest of the codebase, the same treatment can be applied to [`BasicInlineContainer`](https://github.com/swiftlang/swift-markdown/blob/main/Sources/Markdown/Structural%20Restrictions/BasicInlineContainer.swift), the initializers on the Doxygen elements, and more places that were written before `some Protocol` was widespread...